exploration/open-close-expressions.md

```diff
@@ -195,13 +201,40 @@ Markup is not valid in _declarations_ or _selectors_.
 
 * Introduces two new sigils, the pound sign `#` and the forward slash `/`.
 
-* As in HTML, differentiating "open" and "standalone" elements relies on registry information,
-  or in translations matching the structure used in the source
-  A rather clunky `{#foo/}` syntax was considered for explicitly-standalone elements,
-  but this did not reach consensus support.
-
 * In Mustache, the `{{#foo}}`...`{{/foo}}` syntax is used for *control flow* statements rather than printable data.
 
+### Standalone Markup
+
+_Standalone_ markup remains an open question.
+Three alternatives have been suggested:
+
+* Introduce additional syntax inspired by XML: `{#foo/}`.
+
+  This approach encodes the _standalone_ aspect in the data model, and doesn't rely on external sources of truth,
```

Comment:
Not just the data model: it's in the syntax too!

Reply:
This is mentioned in the same sentence, which continues onto the next line: `at the expense of introducing new syntax, which is admittedly a bit clunky.`